homeassistant/components/aladdin_connect/__init__.py

```diff
@@ -1,28 +1,36 @@
 """The aladdin_connect component."""
+import asyncio
+from datetime import timedelta
 import logging
 from typing import Final
 
-from aladdin_connect import AladdinConnectClient
+from AIOAladdinConnect import AladdinConnectClient
+from aiohttp import ClientConnectionError
 
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
 from homeassistant.core import HomeAssistant
-from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
 
 from .const import DOMAIN
 
 _LOGGER: Final = logging.getLogger(__name__)
 
 PLATFORMS: list[Platform] = [Platform.COVER]
+SCAN_INTERVAL = timedelta(seconds=120)
 
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up platform from a ConfigEntry."""
     username = entry.data[CONF_USERNAME]
     password = entry.data[CONF_PASSWORD]
     acc = AladdinConnectClient(username, password)
```

Comment:
We want libraries that use aiohttp to use the Home Assistant aiohttp client session.

https://github.com/home-assistant/core/blob/eb6afd27b312ed2910a1ad32aa20a3f4ebd54cf8/homeassistant/helpers/aiohttp_client.py#L38-L43